packages/mui-material/src/Hidden/withWidth.js

```diff
@@ -54,6 +54,7 @@ const withWidth =
        */
       const keys = theme.breakpoints.keys.slice().reverse();
       const widthComputed = keys.reduce((output, key) => {
+        // TODO: uncomment when compiler enabled // eslint-disable-next-line react-compiler/react-compiler
```

Comment:
Let's use the same comment we use in the rest of the code base

```suggestion
        // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler
```